force-app/main/default/lwc/recentSessions/recentSessions.js

```diff
@@ -73,6 +73,8 @@ export default class RecentSessions extends LightningElement {
 
     HOME = "Home";
     END = "End";
+    SMALL_REGION_WIDTH = 480;
```

Comment:
Noticed this on the call but would these be better at the class level since they are constants?